docs/source/exportdatacode.rst

```diff
@@ -88,11 +76,18 @@ We are now going to use datalad to create a sibling dataset on OSF with name `os
 
     $ datalad create-sibling-osf -s osf OSF_PROJECT_NAME
 
-And finally we are going to copy your data to this new OSF project - collaborators can now clone your OSF repo with versioned data and at the same time there exists a browsable repo containing your code on Github.
+Setting up Github Remote
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can set-up a Github Remote with name `github` and include a publish dependency with OSF - that way, when we publish our dataset to Github, the data files get automatically uploaded to OSF.
 
 .. code-block:: bash
 
-    $ git annex copy . --to osf
+    $ datalad create-sibling-github REPRONAME -s github --github-login GITHUB_NAME --publish-depends osf
+    $ datalad publish --to github
```

Comment:
try `datalad publish . --to github`

Reply:
same result unfortunately

Reply:
seems `publish` didn't pick up, that data actually needs to be transfered. So `.` as the path and/or `--transfer-data all` should do the job.

Reply:
Then just replace with a dedicated publish --to osf followd by publish --to github for now. And then let's merge and release. We can figure that out later.

Reply:
that worked! thx!

Reply:
So just the same publish commands you used earlier, only now at the end and in new order. pbulish-depends will still be configured and find out what goes wrong and fix it up. Will then need only small changes and already lays out how it is meant to be